src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php

```diff
@@ -32,6 +32,11 @@
 abstract class DoctrineType extends AbstractType
 {
     /**
+     * @var DoctrineChoiceLoader[]
+     */
+    private static $choiceLoaders = array();
```

Comment:
That's fine, as the (object hash of the) object manager is part of the cache key